Fixed regression errors

p1r473 opened the pull request:
Hi,
When I had added the delete functionality I accidently broke the browse functionality. Oops!
This fixes it.

alpha-tango-kilo commented:
Looks like you've got conflicts. Once you've resolved that, happy to merge :)

p1r473 commented:
Fixed!

alpha-tango-kilo commented on wd.sh:
```diff
@@ -261,12 +261,12 @@ wd_browse() {
     if [[ -e $wd_remove_output ]]; then
         cat "$wd_remove_output"
-        rm "$wd_remove_output"
+        rm "$wd_remove_output" >/dev/null 2>&1
```
Why are you expecting `rm` to output?

p1r473 replied:
I was getting something about "removing regular file" but then I realized I had rm aliased. I dont seem to be able to reproduce this so Ive removed the pipe

p1r473 replied:
I love how thorough you are!

alpha-tango-kilo reviewed (approved):
Okay, now I'm just curious as to what you've changed that fixes your regression

You've renamed and removed the `local` keyword from a variable declaration. How did that fix the issue?

p1r473 commented:
I also renamed the variable from fzf_command to selected_entry as I had renamed it during testing and was not actually utilizing that new variable name

p1r473 commented:
I didn't remove the local, the local is previously declared
